### DISCUSSION TITLE
[Enhancement] [select] select选择器中filterable筛选选项

### DISCUSSION THREAD

**Discussion by 1393254712** [element-plus/element-plus#18532] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

选择器的筛选选项当我输入内容后 鼠标点击文本框 能不能不要清空输入的内容和收起下拉列表 
而是只聚焦到点击的文本位置 不做其他操作 当点击输入框外部再进行清空和收起列表
这样是不是更符合操作习惯

### Additional comments

_(empty)_

**Comment by 1393254712** [comment]
在这里提有用的吗

**Reply by warmthsea** [reply]
> 在这里提有用的吗

我觉得在可输入搜索选择框时候，已经输入文字再次点击输入框下拉框不收回列表是个很好的想法，有些UI库有这个功能
欢迎你来 PR
